packages/kit/src/components/TxAction/TxActionUnknown.tsx

```
@@ -79,12 +85,19 @@
+  const route =
+    useRoute<RouteProp<IModalSendParamList, EModalSendRoutes.SendConfirm>>();
+  const sourceInfo = route.params?.sourceInfo;
+
   return (
     <TxActionCommonDetailView
       networkId={decodedTx.networkId}
       overview={{
         content: intl.formatMessage({
-          id: ETranslations.transaction__contract_interaction,
+          id:
+            sourceInfo && sourceInfo.origin
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)